mws/sitesmanagement/signals.py

```diff
@@ -18,15 +18,14 @@ def check_main_domain_name(instance, **kwargs):
 @receiver(pre_delete, sender=SiteKey)
 def delete_sshfp_from_dns(instance, **kwargs):
     '''Delete SSHFP records from the DNS using the DNS API when a SiteKey is deleted from the database'''
-    if instance.type != "ED25519":
-        for service in instance.site.services.all():
+    for service in instance.site.services.all():
+        for fptype in SiteKey.FP_TYPES:
+            delete_sshfp(service.network_configuration.name, SiteKey.ALGORITHMS[instance.type],
```

Comment:
this will break trying to delete SHA1 entries that no longer exists in the DNS

Reply:
I wasn't sure about what the ipreg api does with such requests. back to the drawing board on this one...